chinese_tutorial_rvest_httr.Rmd

```diff
@@ -0,0 +1,490 @@
+
+#Chinese Web scraping using R package: rvest & httr tutorial
```

Comment:
Need a single whitespace after the hashtag

Reply:
This problem has been solved, thank you.